lib/generators/data_migration/data_migration_generator.rb

```diff
@@ -28,6 +28,8 @@ def set_local_assigns!
       def migration_base_class_name
         if ActiveRecord.version >= Gem::Version.new("5.0")
           "ActiveRecord::Migration[#{ActiveRecord::Migration.current_version}]"
+        elsif ActiveRecord.version >= Gem::Version.new("5.2")
+          "DataMigrate::MigrationContext"
```

Comment:
Running Rails 5.2 and everything is working properly with the generators. However, doesn't seem like this code path could actually get used.